src/content-handlers/iiif/extensions/uv-aleph-extension/config/config.json

```diff
@@ -110,6 +110,7 @@
         "feedback": "Feedback",
         "fullScreen": "Full Screen",
         "moreInfo": "More Information",
+        "toggleLeftPanel": "Toggle Left Panel",
```

Comment:
I think it's a bug that this file is missing the shorthand variables for translations. This should probably be $toggleLeftPanel, and the other text should be converted as well (maybe that's a separate PR!).